server/src/main/java/org/opensearch/action/admin/indices/stats/IndicesStatsRequest.java

```diff
@@ -103,6 +106,11 @@ public IndicesStatsRequest groups(String... groups) {
         return this;
     }
 
+    @Override
```

Comment:
The request is created across multiple transport & Rest action. Should we create ClusterAdminTask (extending cancellabletask) only for actions where we are implementing cancellation (and default to task in other places) ?